.idea

__pycache__
train_logs/
models/xception-b5690688.pth

